give.spec

```diff
@@ -28,10 +25,12 @@ URL: https://github.com/hpc/%{name}
 
 %build
 %if %{with checks}
+    %{!?give_dir: %define give_dir /usr/give}
```

Comment:
Curious about this. Why check/set in each condition rather than a single time regardless of with checks or without? give_dir isn't dependent on the condition of "checks".

Reply:
I had erroneously thought that there can be only one `if/else` block in the build section since none of the the syntaxes for the condition I initially tried didn't work. I tried a different syntax, and I was able to move it to its own `if/endif` block.